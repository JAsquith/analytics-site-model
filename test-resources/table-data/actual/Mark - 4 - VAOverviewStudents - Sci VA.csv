,Name,Sci VA
1,Adams; William (005001),-2.760
2,Aykroyd; Dan (005008),-2.760
3,Burnett; Carol (005022),-2.090
4,Cash; Johnny (005035),-1.590
5,Davis; Miles (005049),-1.590
6,Derek; Bo (005053),-1.590
7,Hope; Bob (005092),-1.590
8,Judd; Ashley (005103),-1.590
9,Leoni; Tea (005111),-1.590
10,Mancini; Henri (005122),0.660
11,Manson; Shirley (005123),0.090
12,McReady; Mike (005128),0.090
13,Newton; Isaac (005139),0.090
14,Osment; Emily (005144),0.090
15,Shields; Brooke (005174),0.090
16,Simpson; Ashlee (005177),-0.410
17,Sinatra; Frank (005179),1.590
18,Stephens; Steve (005183),3.090
19,Taylor; James (005188),0.090
20,Veirs; Laura (005199),-0.410
21,Weaver; Sigourney (005204),0.090
,Summary,-0.571
,Summary,-0.385
,Summary,-0.777