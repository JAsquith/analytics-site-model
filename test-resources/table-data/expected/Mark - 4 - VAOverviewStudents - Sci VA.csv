,Name,Sci VA
1,Stephens; Steve (005183),3.090
2,Sinatra; Frank (005179),1.590
3,Mancini; Henri (005122),0.660
4,Shields; Brooke (005174),0.090
5,Weaver; Sigourney (005204),0.090
6,Manson; Shirley (005123),0.090
7,Osment; Emily (005144),0.090
8,McReady; Mike (005128),0.090
9,Newton; Isaac (005139),0.090
10,Taylor; James (005188),0.090
11,Veirs; Laura (005199),-0.410
12,Simpson; Ashlee (005177),-0.410
13,Derek; Bo (005053),-1.590
14,Leoni; Tea (005111),-1.590
15,Judd; Ashley (005103),-1.590
16,Cash; Johnny (005035),-1.590
17,Davis; Miles (005049),-1.590
18,Hope; Bob (005092),-1.590
19,Burnett; Carol (005022),-2.090
20,Adams; William (005001),-2.760
21,Aykroyd; Dan (005008),-2.760
,Summary,-0.385
,Summary,-0.571
,Summary,-0.777
